inst/WORDLIST

```diff
@@ -60,21 +64,19 @@ https
 IG
 knitr
 labelled
+linter
+LLC
 MedDRA
+metacore
 metatools
-Metatools
 mmHg
 modularized
 Mosteller
 msec
+na
 OCCDS
-occuring
-param
-Param
-PARAM
+optionality
 paramater
-pharma
-Pharma
```

Comment:
We want to keep the uppercase Pharma.  I believe it is the convention we want to use across documenation.  However, we want pharma to be flagged so that one should be removed

Reply:
Agree, let's stick to "Pharma".